playground/frontend/build.gradle

```diff
@@ -39,9 +39,20 @@ dependencies {
   dockerDependency project(path: playgroundJobServerProject, configuration: "shadow")
 }
 
+task removeBuild  {
```

Comment:
Do we need to add it to the backend as well?

Reply:
nope, i think it's not necessary